src/arduino_iot_cloud/ucloud.py

```diff
@@ -21,13 +21,13 @@ class InvalidStateError(Exception):
 try:
     from arduino_iot_cloud._version import __version__
 except (ImportError, AttributeError):
-    __version__ = "1.3.3"
+    __version__ = "1.5.0"
```

Comment:
```suggestion
    __version__ = "1.4.1"
```
This will be release `1.4.1`